.gitignore

```diff
@@ -0,0 +1,2 @@
+/pkg/

```

Comment:
Because my `go get` puts them there. I think something is wrong in my configuration. I think it's safer to have this here, no?